src/collision/layers.rs

```diff
@@ -69,6 +69,7 @@ impl<L: PhysicsLayer> PhysicsLayer for &L {
 /// ```
 #[derive(Reflect, Clone, Copy, Debug, Deref, DerefMut, Eq, PartialOrd, Ord)]
 #[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "serialize", reflect(Serialize, Deserialize))]
```

Comment:
```suggestion
#[derive(Reflect, Clone, Copy, Debug, Deref, DerefMut, Eq, PartialEq, PartialOrd, Ord)]
#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
#[cfg_attr(feature = "serialize", reflect(Serialize, Deserialize))]
#[reflect(Debug, PartialEq)]
```
The type was missing `PartialEq`